chainercv/extensions/vis_report/detection_vis_report.py

```diff
@@ -7,10 +7,10 @@
 from chainercv.visualizations.vis_bbox import vis_bbox
 
 try:
-    from matplotlib import pyplot as plot
+    import matplotlib  # NOQA
     _available = True
 
-except ImportError:
+except (ImportError, TypeError):
```

Comment:
Are there any cases that `TypeError` is raised?

Reply:
I copied it from Chainer.

`TypeError` is sometimes called when backend is not imported properly.

###### related

https://github.com/chainer/chainer/pull/2482

https://stackoverflow.com/questions/31328436/typeerror-constructor-returned-null-while-importing-pyplot-in-ssh

Reply:
I see.